### PR TITLE
There is now a user switch to change between notification styles

### PR DESCRIPTION
**Pull request by annando**
You can now decide between condensed and detailed. See discussion here: https://forum.friendi.ca/display/535de4194159f9864538878389872488

### CODE REVIEW

File: mod/ping.php [hunk 1]
```diff
@@ -477,7 +478,11 @@ function ping_get_notifications($uid)
 
 			if ($notification["visible"] && !$notification["spam"] &&
 				!$notification["deleted"] && !is_array($result[$notification["parent"]])) {
-				$result[$notification["parent"]] = $notification;
+				if (PConfig::get(local_user(), 'system', 'detailed_notif')) {
```
Comment (rabuzarus) [inline mod/ping.php]: Can you please add a comment that this part is for showing comments + like notifs (detailed_notif is not that self explaining. So a comment would be useful)

Comment (annando) [reply mod/ping.php]: Yeah, you are right.

File: mod/settings.php [hunk 2]
```diff
@@ -504,14 +516,15 @@ function settings_post(App $a) {
...
-	if(strlen($err)) {
+	if (strlen($err)) {
 		notice($err . EOL);
 		return;
 	}
 
-	if($timezone != $a->user['timezone']) {
-		if(strlen($timezone))
+	if ($timezone != $a->user['timezone']) {
+		if (strlen($timezone)) {
```
Comment (MrPetovan) [inline mod/settings.php]: This can be condensed in a single condition.

**Comment by MrPetovan** [comment]
Wait, there are at least two different screens where notifications appear, and this only affects the notification pop-up that doesn’t show on the mobile version.

What about the standalone notifications page, can we condense the notifications as well there?

**Comment by annando** [comment]
I never had done any changes to the other notifications page, only to ```mod/ping.php```.

File: mod/settings.php (hunk 2 above)
Comment (annando) [reply mod/settings.php]: Done.

**Comment by MrPetovan** [comment]
I know you didn't change the other notifications page, that's why I'm asking about it. It used to be
- Pop-up: Condensed notifications
- Page: Full notifications

Now that the pop-up is getting a setting switch to get either, I was wondering if it made sense to apply it to the notification page as well. I certainly could use it.

**Comment by annando** [comment]
If you want to do it, feel free.

**Comment by MrPetovan** [comment]
Of course I can, provided enough time, my question was if it made sense to do it at all, or if the standalone notification page had a specific philosophy.

**Comment by annando** [comment]
I never really worked with that page and I think that it is another PR.

**Comment by MrPetovan** [comment]
Of course, it wasn't a call for you to do anything else in this PR.

**Comment by annando** [comment]
You could file an issue for that. So that we possibly can do this on the Hackathon.

**Comment by MrPetovan** [comment]
I will, thanks.

**Comment by hfroese** [comment]
Works perfect. Thanks 👍

**Pull request merged**